scala-package/native/src/main/native/ml_dmlc_mxnet_native_c_api.cc

```diff
@@ -1711,3 +1714,435 @@ JNIEXPORT jint JNICALL Java_ml_dmlc_mxnet_LibInfo_mxRtcFree
   int ret = MXRtcFree(handle);
   return ret;
 }
+
+JNIEnv *globalEnv;
+std::unordered_map<std::string, jobject> globalOpPropMap;
+std::unordered_map<std::string, jobject> globalOpMap;
```

Comment:
what I actually tried to implement is that because when those callback function was called, it was called in C++ side, then inside the callback function it needs to find the specific java object then
it can call the java object function, so I store the object in a map during initialization, then retrieve them with their name.